app/src/main/java/is/xyz/mpv/MPVFilePickerFragment.java

```diff
@@ -22,7 +22,9 @@ public void onClickCheckable(@NonNull View v, @NonNull FileViewHolder vh) {
 
     @Override
     public boolean onLongClickCheckable(@NonNull View v, @NonNull DirViewHolder vh) {
-        mListener.onDirPicked(vh.file);
+        if (vh.file.isDirectory()) {
```

Comment:
since vh is a `DirViewHolder`, I'd expect it to only hold directories
does this assumption not hold?

Reply:
Does on my Pixel 2 but doesn't on my Xiaomi so I don't have the slightest idea what is up...

Will rebase this patch out along with the external directory change.